groups/hsp-iit.yml

```diff
@@ -30,6 +30,7 @@ hsp-iit/purchases:
   - "drdanz"
   - "GiuliaP"
   - "fbottarel"
+  - "xEnVrE"
```

Comment:
```suggestion
  - "Arya07"
```

Let's shortcut. We can always fix it later if required.

Reply:
It was for @Arya07 correct.